content/v2.0/query-data/flux/fill.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Fill null values in data
+seotitle: Fill null values in data
+list_title: Fill null values
+description: >
+  Use the [`fill()` function](/v2.0/reference/flux/stdlib/built-in/transformations/fill/)
+  to replace _null_ values in query results.
+weight: 210
+menu:
+  v2_0:
+    parent: Query with Flux
+    name: Fill
+v2.0/tags: [query, fill]
+list_code_example: |
+  ```js
+  data
+    |> fill(usePrevious: true)
+  ```
+---
+
+Use the [`fill()` function](/v2.0/reference/flux/stdlib/built-in/transformations/fill/)
+to replace _null_ values in query results.
```

Comment:
Use the [`fill()` function](/v2.0/reference/flux/stdlib/built-in/transformations/fill/)
to replace _null_ values in query results with:
- Previous non-null values
- Specified value